sources/Inputs/include/Configuration.h

```diff
@@ -98,6 +98,42 @@ class Configuration {
     return dsoVoltageLevel_;
   }
 
+  /**
+   * @brief Get the Time at which the simulation will start
+   *
+   * @returns the parameter value
+   */
+  float getStartTime() const {
```

Comment:
The three times (start, stop, time of event) should be `float` or `double`. Why start and stop are `float` and time of event `double`?.